ibis/expr/operations.py

```diff
@@ -59,6 +59,18 @@ def _pp(x):
 
         return '{}({})'.format(opname, ', '.join(pprint_args))
 
+    def __getstate__(self):
```

Comment:
Can you add docstring for these two method describing what you are doing and why?